src/components/chat/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { type NextPage } from "next";
+
+const Chat: NextPage = () => {
```

Comment:
I was thinking, would it be better if we use the React.FC type for this component instead of NextPage? Could you kindly confirm if this change is indeed suitable?